gammapy/estimators/sensitivity.py

```diff
@@ -61,13 +63,12 @@ def estimate_min_excess(self, dataset):
         n_off = dataset.counts_off.data
 
         stat = WStatCountsStatistic(
-            n_on=np.ones_like(n_off), n_off=n_off, alpha=dataset.alpha.data
```

Comment:
Why was n_on chose as `1` previously? In the absence of a signal estimate, `n_on` should be as expected from the off counts, no?

Reply:
The information `n_on` is not used in this case. So I guess it's just some place-holder value. One could maybe use `NaN` to make it more clear...

Reply:
I see.
But should this not call - `WStatCountsStatistic._excess_matching_significance` ?

Reply:
Also, `SensitivityEstimator.estimate_min_excess` supposedly takes a `SpectrumDataset` as input but uses `WStatCountsStatistic`.
Should it be adapted to use either `cash` or `wstat` as per the case, or does this work only for `SpectrumDatasetOnOff`?